src/main/java/com/vmware/vipclient/i18n/messages/service/ProductService.java

```diff
@@ -91,4 +87,18 @@ public ProductService(BaseDTO dto) {
         }
         return locales;
     }
+
+    public Set<Locale> getSupportedLocales() {
```

Comment:
```suggestion
    public Set<Locale> getSupportedLocaleSet() {
```

Reply:
We actually only need 1 method. I will use: `public Set<Locale> getSupportedLocales() `to replace the following 2 signatures:

- public Set<Locale> getSupportedLocales()
- public List<String> getSupportedLocales()